src/components/taskItem/TaskItem.tsx

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import styles from './TaskItem.module.css';
+import { tryApiRequestCatchError } from "../../utility/api";
+
+type TaskItemProps = {
+    id: number;
+    title: string;
+    status: string;
+    getAllTasks: () => Promise<void>;
+}
+
+export const TaskItem = ({id, status, title, getAllTasks}: TaskItemProps) => {
+    let taskBody = {
+        status: "closed",
+        id: id
```

Comment:
This can be written only `id`.
When the object property and the constant have the same name.
```
const name = "a-name";
const obj = {name: name};

// is equal to

const obj = {name};
```